xml/core_registers.xml

```diff
@@ -22,7 +32,15 @@
             When 1, {\tt ebreak} instructions in User/Application Mode enter
             Debug Mode.
         </field>
-        <field name="0" bits="11" access="R" reset="0" />
+        <field name="intdisable" bits="11" access="R/W" reset="0">
+            0: Interrupts are handled as usual.
+
+            1: Interrupts are disabled. This bit supersedes mstatus.ie.
+
+            This feature may not exist in implementations from early in the
```

Comment:
Also, I can't think of a case where you would actually want interrupts during debug mode. Everything about how MEPC etc would get updated is unclear. You don't want to be getting interrupts in the middle of executing program buffer, for example. 

During single step is a different story, because you leave debug mode during single step.

Do you have an application in mind when you'd actually want interrupts enabled *during debug mode*?